---
#I am going to do the barcoding page now
#Install nbaR

# indentations are critical in YAML headers, be careful when editing this section

title: "DNA barcoding in South Africa" # Compulsory
subtitle: "Genetic diversity tools" 

title-block-banner: "#99BD41" #Background colour of the title block: Apply component colours where necessary - find hex codes in _brand.yml
title-block-banner-color: white # Header text colour: DO NOT CHANGE

# List authors here in the order you want them displayed in the title block
# See examples in README
author:
  # Copy this block and add details for each author of your page. Make sure you maintain indentations exactly as they are here
  - name:
      given: Mahlatse #Add author's first name here. Abbreviate the middle name
      family: Kgatla #Add author's surname here
    affiliations:
      - ref: af1 # Ref here must match the id of one of the affiliations listed below
      # If an author has more than one affiliation, add more - ref: lines here
  # Copy this block and add details for each author of your page. Make sure you maintain indentations exactly as they are here
  - name:
      given: Janine #Add author's first name here. Abbreviate the middle name
      family: Baxter #Add author's surname here
    orcid: 0000-0002-6050-1144 # Author's orcid ID. Delete this line if author is not on orcid
    affiliations:
      - ref: af1 # Ref here must match the id of one of the affiliations listed below
      # If an author has more than one affiliation, add more - ref: lines here

  # Copy this block and add details for each author of your page. Make sure you maintain indentations exactly as they are here
  - name:
      given: Monica #Add author's first name here. Abbreviate the middle name
      family: Mwale #Add author's surname here
    orcid: 0000-0003-2180-8917 # Author's orcid ID. Delete this line if author is not on orcid
    affiliations:
      - ref: af1 # Ref here must match the id of one of the affiliations listed below
      # If an author has more than one affiliation, add more - ref: lines here

affiliations:
# List affiliations here and number them in the order that they are associated with the list of authors above. If multiple authors have the same affiliation, only list the affiliation once here.
# Write out institutional names in full
# See README for common examples - they can be copied and pasted here
# Make sure you maintain indentations exactly as they are here
  - id: af1 # Use this as the affiliation ref with authors above
    number: 1 # This number is used to associate authors with affiliations
    name: South African National Biodiversity Institute


# This date determines the PUBLISHED date in the title block
date: last-modified

# Remember to rename the default .bib file and change the name here
bibliography: genetics.bib

# This section sets up the page citation format. DO NOT EDIT
citation:
  type: webpage
  container-title: National Biodiversity Assessment 2025
  publisher: South African National Biodiversity Institute
  url: http://nba.sanbi.org.za/
# Do not edit or change any of the following entries
appendix-cite-as: false
google-scholar: true
---

## Understanding South Africa’s biodiversity through DNA barcoding

South Africa’s rich biodiversity spans terrestrial, freshwater, and marine ecosystems, encompassing complex community assemblages of organisms. This diversity is a crucial national asset, underpinning strategic objectives across conservation, sustainable resource management, and socio-economic development[@skownoNationalBiodiversityAssessment2018]. Understanding and documenting biodiversity is therefore essential for informed decision-making, guiding conservation strategies, sectoral policies, and sustainable use of our natural resources to address challenges such as poverty, inequality and unemployment [@cbdConventionBiologicalDiversity].

Despite this, many South African ecosystems and species remain understudied, even as they face growing pressures like habitat loss and fragmentation, unsustainable use, climate change, biological invasions and illegal trade. For numerous species, conservation status remains data deficient or unassessed, complicating efforts to evaluate and mitigate threats. These knowledge gaps are compounded by incomplete taxonomic data, including uncertainties in species classification, identification and distribution across bioregions.

An important tool that can assist in addressing these challenges is DNA barcoding, which enables rapid and accurate species identification using standardised genetic markers [@hebertBiologicalIdentificationsDNA2003; @odahUnlockingGeneticCode2023]. DNA barcoding improves understanding of species diversity across ecosystems and biomes [@antilDNABarcodingEffective2023; @ratnasinghamBOLDBarcodeLife2007], supports the updating of national species checklists, and helps fill taxonomic gaps in priority groups. It complements traditional taxonomy by reliably delimiting species and identifying Operational Taxonomic Units (OTUs), including cryptic, closely related or morphologically similar populations [@ratnasinghamDNABasedRegistryAll2013; @shethDNABarcodingTraditional2017]. DNA barcoding also has practical applications in environmental DNA (eDNA) monitoring of ecosystems, tracking invasive species [@farrellEnvironmentalDNATracking; @luAdvancesEnvironmentalDNA2024; @schenekarUnveilingVertebrateBiodiversity2025] and forensic identification of illegal wildlife trade in South Africa [@baxterDNABarcodingSouthern2024; @mwaleDNABarcodingIdentification2021].

DNA barcoding relies on short, standardised DNA regions for identification – e.g., mitochondrial cytochrome oxidase subunit I (*COI*) for many animals, maturase K (*matK*0 and ribulose-1,5-bisphosphate carboxylase/oxygenase (*RbcL*) for plants, and Internal Transcribed Spacer (*ITS)* for fungi [@mathurDNABarcodingIdentification2021].

## South African Barcode of Life (SABOL)

The [**South African Barcode of Life (SABOL)**](https://fbip.co.za/about/sanbi-gbif/) is the national node of the International Barcode of Life Consortium (iBOL). SABOL is funded by the Department of Science, Technology and Innovation (DSI) through the Foundational Biodiversity Information Programme (FBIP). SABOL promotes DNA barcoding research, addressing gaps in species reference libraries, and consolidates scattered biodiversity data into one comprehensive national resource. DNA barcoding strengthens biodiversity assessments, particularly for cryptic and understudied taxa.

Between 2013 and 2022, the FBIP funded 124 DNA barcoding-related projects totalling over R60 million (@fig-fbip_investment). These efforts align with both international and national obligations, including the United Nations Convention on Biological Diversity (CBD), the Intergovernmental Platform on Biodiversity and Ecosystem Services (IPBES), and South Africa's National Environmental Management: Biodiversity Act (NEM:BA).

![Funds (in millions of South African Rands) awarded per year to biodiversity research through the Foundational Biodiversity Information Programme (FBIP) between 2013 and 2022. Plot adapted from Mahlatse et al. under review.](imgs/fbip_investment.png){#fig-fbip_investment fig-align="center" width="600"}

A key component of DNA barcoding is the availability of a comprehensive reference library of expertly identified specimens. The [**Barcode of Life Database (BOLD)**](https://boldsystems.org/) serves as a trusted repository for these sequences, ensuring that barcodes meet rigorous quality standards and are derived from taxonomically verified specimens. By providing a reliable foundation for species identification, BOLD underpins SABOL and other national barcoding initiatives, enabling accurate biodiversity assessments and monitoring.

## South African barcoding efforts to date

Over the past decade, South Africa has made significant progress in expanding its faunal DNA barcoding database, particularly for insects and other invertebrates. A 2016 review of DNA barcoding reported that South Africa had contributed approximately 48 000 faunal DNA barcodes to BOLD, representing approximately 2.3% of all known species globally, at the time[@dasilvaReviewDecadeDNA2016].

Since 2016, contributions to the reference library have grown substantially, with numerous South African researchers and institutions submitting data on indigenous species. As of 28 October 2025, BOLD contains 786 480 sequences (multiple gene sequences for some records), representing 776 512 specimens from South African localities. Approximately, 93% (725 590) of these records are assigned to 59 606 Barcode Index Numbers (BINs), which represent OTUs. However, only 94 010 specimen records have been identified to species level, representing 16 425 species (@fig-taxon-bin). This highlights ongoing needs for taxonomic capacity and expertise.

This milestone reflects the collaborative efforts of 284 research institutions, demonstrating the commitment of South Africa's biodiversity research community to building a comprehensive DNA barcode species reference library. Despite this progress, challenges remain, including incomplete geographical coverage, taxonomic gaps, limited taxonomic expertise, and the underutilisation of barcode data in conservation and management frameworks [@dasilvaReviewDecadeDNA2016].

![BIN (Barcode Index Numbers) to species ratio regression analysis. The relationship between the number of BINs assigned to records and the number of species morphologically identified and assigned to species name, with the orange line indicating the 1:1 ratio (BIN to Species). Plot adapted from Mahlatse et al. under review.](imgs/taxon-vs-bin.png){#fig-taxon-bin fig-align="center" width="600"}

## Future directions and recommendations

A 2025 gap analysis [@mahlatse-underreview] indicates that South Africa's reference library still has taxonomic and geographic gaps, particularly in priority taxa and under-sampled regions (@fig-distribution). The BIN system relies on multiple sequences per species that capture intraspecific variation across distributions and to improve phylogenetic resolution [@ratnasinghamDNABasedRegistryAll2013]. To maximise its effectiveness, future sampling should focus on:

-   **entire species' ranges**, especially for known species that have not yet been barcoded;

-   **sparsely sampled areas**, including biodiversity hotpots and threatened ecosystems;

-   **understudied taxonomic groups**, ensuring priority taxa are adequately represented.

![Proportional representation of taxon groups with BINs in each South African province. Taxon group ‘Fungi’ is excluded because it had no BIN records on BOLD. Category “Other Animals” are species that do not fit into the outlined taxon groups. Category “Other Organisms” are organisms other than animal, plant or fungi (e.g., Protista). Numbers indicate the number of BINs. Map adapted from Mahlatse et al. under review.](imgs/distribution.png){#fig-distribution fig-align="center" width="650"}

Currently, a significant number of South African records on BOLD remain private while researchers analyse them (@fig-public-private). While understandable, this can create redundancies and slow the growth of the publicly accessible reference library.

![Number of South African records on BOLD broken down by public (published) and private (unpublished) records. Graph adapted from Mahlatse et al. under review.](imgs/private-public.png){#fig-public-private width="800"}

To address these challenges, it is recommended that DNA barcoding efforts be coordinated and collaborative, ensuring national resources are used efficiently and research gaps are strategically targeted. Establishing a directory of existing barcoding projects and project leaders, alongside regular SABOL-led community meetings and conferences, will strengthen communication and collaboration among institutions.

## Recommended citation

```{r}
#| output: asis
#| label: lst-citation-code

library(nbaR) 
library(knitr)
library(rmarkdown)

# Code for fetching the yaml metadata
# Note that this only runs when the .qmd file is rendered, therefore the outputs of this code block will return no results if you try to run it within R - that does not mean it is not working

meta <- knitr::opts_knit$get("rmarkdown.pandoc.to")  # forces knitr to load metadata
meta <- rmarkdown::metadata  # full YAML is now in `meta`

nba_citation(meta)

```

## References

::: {#refs}
:::
