<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.25">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Mahlatse Kgatla">
<meta name="author" content="Janine Baxter">
<meta name="author" content="Monica Mwale">
<meta name="dcterms.date" content="2025-11-11">

<title>DNA barcoding in South Africa</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../imgs/sanbi-icon.png" rel="icon" type="image/png">
<script src="../../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../../site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-7b89279ff1a6dce999919e0e67d4d9ec.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap-873486f3db474cc0cfb34b8517f191db.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<style>

      .quarto-title-block .quarto-title-banner h1,
      .quarto-title-block .quarto-title-banner h2,
      .quarto-title-block .quarto-title-banner h3,
      .quarto-title-block .quarto-title-banner h4,
      .quarto-title-block .quarto-title-banner h5,
      .quarto-title-block .quarto-title-banner h6
      {
        color: white;
      }

      .quarto-title-block .quarto-title-banner {
        color: white;
background: #99BD41;
      }
</style>
<style>
@media (min-width: 992px) { /* because the bread crumbs are only on the banner on large screens */
  .breadcrumb li a {
    color: white !important;
  }
}
</style>


<meta name="citation_title" content="DNA barcoding in South Africa">
<meta name="citation_author" content="Mahlatse Kgatla">
<meta name="citation_author" content="Janine Baxter">
<meta name="citation_author" content="Monica Mwale">
<meta name="citation_publication_date" content="2025-11-11">
<meta name="citation_cover_date" content="2025-11-11">
<meta name="citation_year" content="2025">
<meta name="citation_online_date" content="2025-11-11">
<meta name="citation_fulltext_html_url" content="http://nba.sanbi.org.za/">
<meta name="citation_language" content="en">
<meta name="citation_journal_title" content="National Biodiversity Assessment 2025">
<meta name="citation_publisher" content="South African National Biodiversity Institute">
<meta name="citation_reference" content="citation_title=Advancing land-sea integration for ecologically meaningful coastal conservation and management;,citation_author=Linda R. Harris;,citation_author=Mariel Bessinger;,citation_author=Anisha Dayaram;,citation_author=Stephen Holness;,citation_author=Stephen Kirkman;,citation_author=Tamsyn-Claire Livingstone;,citation_author=Amanda T. Lombard;,citation_author=Melanie Lück-Vogel;,citation_author=Maya Pfaff;,citation_author=Kerry J. Sink;,citation_author=Andrew L. Skowno;,citation_author=Lara Van Niekerk;,citation_publication_date=2019-09;,citation_cover_date=2019-09;,citation_year=2019;,citation_fulltext_html_url=http://dx.doi.org/10.1016/j.biocon.2019.06.020;,citation_doi=10.1016/j.biocon.2019.06.020;,citation_volume=237;,citation_language=en;,citation_journal_title=Biological Conservation;">
<meta name="citation_reference" content="citation_title=Megadiversity: Earth’s biologically wealthiest nations;,citation_author=R. A. Mittermeier;,citation_author=P. Robles-Gil;,citation_author=C. G. Mittermeier;,citation_publication_date=1997;,citation_cover_date=1997;,citation_year=1997;">
<meta name="citation_reference" content="citation_title=Indicators;,citation_author=J. R. Wilson;,citation_author=B. W. Van Wilgen;,citation_author=K. Faulkner;,citation_author=D. Richardson;,citation_author=S. Rahlao;,citation_author=T. Zengeya;,citation_editor=B. W. Van Wilgen;,citation_editor=J. R. Wilson;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_inbook_title=The status of biological invasions and their management in south africa in 2017;">
<meta name="citation_reference" content="citation_title=National terrestrial carbon sinks assessment: Technical report;,citation_author=Graham Von Maltitz;,citation_author=Bob Scholes;,citation_author=Marc Pienaar;,citation_author=Claire Davis-Reddy;,citation_author=Tony Knowles;,citation_author=Liesl Wiese;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://csa.environment.gov.za/technical-reports#h.atp56x6v335w;,citation_technical_report_institution=Department of Environment, Forestry and Fisheries;">
<meta name="citation_reference" content="citation_title=Chapter 3: Direct and indirect drivers of land degradation and restoration;,citation_author=N. N. Barger;,citation_author=T. A. Gardner;,citation_author=M. Sankaran;,citation_author=J. Belnap;,citation_author=L. Broadhurst;,citation_author=V. Brochier;,citation_author=F. Isbell;,citation_author=P. Meyfroidt;,citation_author=F. Moreira;,citation_author=T. M. Nieminen;,citation_author=T. Okuro;,citation_author=R. R. Rodrgiues;,citation_author=V. Saxena;,citation_author=M. C. Ross;,citation_editor=L. Montanarella;,citation_editor=R. Scholes;,citation_editor=A. Brainich;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_inbook_title=The IPBES assessment report on land degradation and restoration;">
<meta name="citation_reference" content="citation_title=South african national biodiversity assessment 2018 technical report volume 1: Terrestrial realm;,citation_editor=A. L. Skowno;,citation_editor=D. C. Raimondo;,citation_editor=C. J. Poole;,citation_editor=B. Fizzotti;,citation_editor=J. A. Slingsby;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;">
<meta name="citation_reference" content="citation_title=Investigating the direct and indirect drivers of land use and land cover change from agriculture to acacia plantations in banja district, awi zone, ethiopia;,citation_author=Engidayehu Melaku;,citation_author=Asabeneh Alemayehu;,citation_author=Kassa Chanie;,citation_publication_date=2024-06-08;,citation_cover_date=2024-06-08;,citation_year=2024;,citation_fulltext_html_url=http://dx.doi.org/10.37284/eajab.7.1.1973;,citation_issue=1;,citation_doi=10.37284/eajab.7.1.1973;,citation_volume=7;,citation_journal_title=East African Journal of Agriculture and Biotechnology;">
<meta name="citation_reference" content="citation_title=Butterflies and Moths database home page;,citation_author=The Natural History Museum;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_fulltext_html_url=http://www.nhm.ac.uk/our-science/data/butmoth;">
<meta name="citation_reference" content="citation_title=Biomes and bioregions of Southern Africa;,citation_author=M. C. Rutherford;,citation_author=L. Mucina;,citation_author=L. W Powrie;,citation_editor=L. Mucina;,citation_editor=M. C. Rutherford;,citation_publication_date=2006;,citation_cover_date=2006;,citation_year=2006;,citation_conference_title=The vegetation of South Africa, Lesotho and Swaziland;,citation_conference=South African National Biodiversity Institute;,citation_series_title=Strelitzia 19;">
<meta name="citation_reference" content="citation_title=Multinational evaluation of genetic diversity indicators for the Kunming-Montreal Global Biodiversity Framework;,citation_author=A. Mastretta-Yanes;,citation_author=J. M. Silva;,citation_author=C. E. Grueber;,citation_author=L. Castillo-Reina;,citation_author=V. Köppä;,citation_author=B. R. Forester;,citation_author=W. C. Funk;,citation_author=F. Heuertz;,citation_author=R. Jordan;,citation_author=J. Mergeay;,citation_author=I. Paz-Vinas;,citation_author=V. J. Rincon-Parra;,citation_author=M. A. Rodriguez-Morales;,citation_author=L. Arredondo-Amezcua;,citation_author=G. Brahy;,citation_author=M. DeSaix;,citation_author=L. Durkee;,citation_author=A. Hamilton;,citation_author=M. E. Hunter;,citation_author=A. Koontz;,citation_author=I. Lang;,citation_author=M. C. Latorre-Cárdenas;,citation_author=T. Latty;,citation_author=A. Llanes-Quevedo;,citation_author=A. J. MacDonald;,citation_author=M. Mahoney;,citation_author=C. Miller;,citation_author=J. F. Ornelas;,citation_author=S. Ramírez-Barahona;,citation_author=E. Robertson;,citation_author=I-R.M. Russo;,citation_author=M. A. Santiago;,citation_author=R. E. Shaw;,citation_author=G. M. Shea;,citation_author=P. Sjögren-Gulve;,citation_author=E. Suzuki Spence;,citation_author=T. Stack;,citation_author=S. Suárez;,citation_author=A. Takenaka;,citation_author=H. Thurfjell;,citation_author=S. Turbek;,citation_author=M. Merwe;,citation_author=F. Visser;,citation_author=A. Wegier;,citation_author=G. Wood;,citation_author=E. Zarza;,citation_author=L. Laikre;,citation_author=S. Hoban;,citation_publication_date=2024-09;,citation_cover_date=2024-09;,citation_year=2024;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/ele.14461;,citation_doi=10.1111/ele.14461;,citation_volume=27;,citation_language=en;,citation_journal_title=Ecology Letters;">
<meta name="citation_reference" content="citation_title=Where lineages persist: phylogenetic patterns and evolutionary vulnerability in South Africa’s herpetofauna;,citation_author=A. Nieto Lawrence;,citation_author=J. Šmíd;,citation_author=R. Gumbs;,citation_author=K. A. Tolley;,citation_publication_date=00;,citation_cover_date=00;,citation_year=0;,citation_language=en;,citation_journal_title=Diversity and Distributions;">
<meta name="citation_reference" content="citation_title=Indicators to monitor the status of the tree of life;,citation_author=R. Gumbs;,citation_author=A. Chaudhary;,citation_author=B. H. Daru;,citation_author=D. P. Faith;,citation_author=F. Forest;,citation_author=C. L. Gray;,citation_author=A. Kowalska;,citation_author=W. Lee;,citation_author=R. Pellens;,citation_author=S. Pipins;,citation_author=L. J. Pollock;,citation_author=J. Rosindell;,citation_author=R. A. Scherson;,citation_author=N. R. Owen;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_fulltext_html_url= https://doi.org/10.1111/cobi.14138;,citation_doi=10.1111/cobi.14138;,citation_volume=37;,citation_language=en;,citation_journal_title=Conservation Biology;">
<meta name="citation_reference" content="citation_title=DNA barcoding of southern African mammal species and construction of a reference library for forensic application;,citation_author=Janine Rose Baxter;,citation_author=Antoinette Kotze;,citation_author=Marli Bruyn;,citation_author=Katlego Matlou;,citation_author=Kim Labuschagne;,citation_author=Monica Mwale;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_issue=10;,citation_volume=67;,citation_journal_title=Genome;">
<meta name="citation_reference" content="citation_title=A septennium review of wildlife forensic DNA analysis in South Africa;,citation_author=Marli De Bruyn;,citation_author=Desiré Lee Dalton;,citation_author=Cindy K Harper;,citation_author=Mamadi Theresa Sethusa;,citation_publication_date=2025;,citation_cover_date=2025;,citation_year=2025;,citation_journal_title=Forensic Science International: Genetics;">
<meta name="citation_reference" content="citation_title=The power of resolution of microsatellite markers and assignment tests to determine the geographic origin of cheetah (Acinonyx jubatus) in southern Africa;,citation_author=Antoinette Kotze;,citation_author=Karen Ehlers;,citation_author=DC Cilliers;,citation_author=JP Grobler;,citation_publication_date=2008;,citation_cover_date=2008;,citation_year=2008;,citation_issue=6;,citation_volume=73;,citation_journal_title=Mammalian Biology;">
<meta name="citation_reference" content="citation_title=Triangulating the provenance of African elephants using mitochondrial DNA;,citation_author=Yasuko Ishida;,citation_author=Nicholas J Georgiadis;,citation_author=Tomoko Hondo;,citation_author=Alfred L Roca;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_issue=2;,citation_volume=6;,citation_journal_title=Evolutionary Applications;">
<meta name="citation_reference" content="citation_title=Tortoise forensics: Conservation genetics of the leopard tortoise Stigmochelys pardalis in southern Africa;,citation_author=Urban Dajcman;,citation_author=Margaretha Hofmeyr;,citation_author=Flora Ihlow;,citation_author=others;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;">
<meta name="citation_reference" content="citation_title=Isolation and characterization of 30 STRs in Temminck’s ground pangolin (Smutsia temminckii) and potential for cross amplification in other African species;,citation_author=Zelda Du Toit;,citation_author=Desiré L Dalton;,citation_author=Morné Du Plessis;,citation_author=Raymond Jansen;,citation_author=J Paul Grobler;,citation_author=Antoinette Kotzé;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=1;,citation_volume=99;,citation_journal_title=Journal of Genetics;">
<meta name="citation_reference" content="citation_title=Development and validation of a novel forensic STR multiplex assay for blue (Anthropoides paradiseus), wattled (Bugeranus carunculatus), and grey-crowned crane (Balearica regulorum);,citation_author=Marli De Bruyn;,citation_author=Desiré Lee Dalton;,citation_author=Monica Mwale;,citation_author=Karen Ehlers;,citation_author=Antoinette Kotze;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_volume=73;,citation_journal_title=Forensic Science International: Genetics;">
<meta name="citation_reference" content="citation_title=Unlocking the potential of a validated single nucleotide polymorphism array for genomic monitoring of trade in cheetahs (Acinonyx jubatus);,citation_author=Michelle Magliolo;,citation_author=Stefan Prost;,citation_author=Pablo Orozco-terWengel;,citation_author=Pamela Burger;,citation_author=Anna S Kropff;,citation_author=Antoinette Kotze;,citation_author=J Paul Grobler;,citation_author=Desire Lee Dalton;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=1;,citation_volume=48;,citation_journal_title=Molecular Biology Reports;">
<meta name="citation_reference" content="citation_title=RhODIS®(The Rhinoceros DNA Index System): The application of simple forensic and genetic tools help conserve African rhinoceros;,citation_author=Cindy Kim Harper;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_inbook_title=Wildlife Biodiversity Conservation: Multidisciplinary and Forensic Approaches;">
<meta name="citation_reference" content="citation_title=National Environmental Management Act 10 of 2004: Amendments to the regulations on Bioprospecting, Access and Benefit-Sharing notice 447 of 2015;,citation_author=Government Africa;">
<meta name="citation_reference" content="citation_title=DNA barcoding, an effective tool for species identification: A review;,citation_abstract=DNA barcoding is a powerful taxonomic tool to identify and discover species. DNA barcoding utilizes one or more standardized short DNA regions for taxon identification. With the emergence of new sequencing techniques, such as Next-generation sequencing (NGS), ONT MinION nanopore sequencing, and Pac Bio sequencing, DNA barcoding has become more accurate, fast, and reliable. Rapid species identification by DNA barcodes has been used in a variety of fields, including forensic science, control of the food supply chain, and disease understanding. The Consortium for Barcode of Life (CBOL) presents various working groups to identify the universal barcode gene, such as COI in metazoans; rbcL, matK, and ITS in plants; ITS in fungi; 16S rRNA gene in bacteria and archaea, and creating a reference DNA barcode library. In this article, an attempt has been made to analyze the various proposed DNA barcode for different organisms, strengths &amp;amp;amp; limitations, recent advancements in DNA barcoding, and methods to speed up the DNA barcode reference library construction. This study concludes that constructing a reference library with high species coverage would be a major step toward identifying species by DNA barcodes. This can be achieved in a short period of time by using advanced sequencing and data analysis methods.;,citation_author=Sandeep Antil;,citation_author=Jeeva Susan Abraham;,citation_author=S. Sripoorna;,citation_author=Swati Maurya;,citation_author=Jyoti Dagar;,citation_author=Seema Makhija;,citation_author=Pooja Bhagat;,citation_author=Renu Gupta;,citation_author=Utkarsh Sood;,citation_author=Rup Lal;,citation_author=Ravi Toteja;,citation_publication_date=2023-01;,citation_cover_date=2023-01;,citation_year=2023;,citation_issue=1;,citation_doi=10.1007/s11033-022-08015-7;,citation_issn=1573-4978;,citation_pmid=36308581;,citation_volume=50;,citation_language=en-US;,citation_journal_title=Molecular Biology Reports;">
<meta name="citation_reference" content="citation_title=DNA barcoding, an effective tool for species identification: A review;,citation_abstract=DNA barcoding is a powerful taxonomic tool to identify and discover species. DNA barcoding utilizes one or more standardized short DNA regions for taxon identification. With the emergence of new sequencing techniques, such as Next-generation sequencing (NGS), ONT MinION nanopore sequencing, and Pac Bio sequencing, DNA barcoding has become more accurate, fast, and reliable. Rapid species identification by DNA barcodes has been used in a variety of fields, including forensic science, control of the food supply chain, and disease understanding. The Consortium for Barcode of Life (CBOL) presents various working groups to identify the universal barcode gene, such as COI in metazoans; rbcL, matK, and ITS in plants; ITS in fungi; 16S rRNA gene in bacteria and archaea, and creating a reference DNA barcode library. In this article, an attempt has been made to analyze the various proposed DNA barcode for different organisms, strengths &amp;amp;amp; limitations, recent advancements in DNA barcoding, and methods to speed up the DNA barcode reference library construction. This study concludes that constructing a reference library with high species coverage would be a major step toward identifying species by DNA barcodes. This can be achieved in a short period of time by using advanced sequencing and data analysis methods.;,citation_author=Sandeep Antil;,citation_author=Jeeva Susan Abraham;,citation_author=S. Sripoorna;,citation_author=Swati Maurya;,citation_author=Jyoti Dagar;,citation_author=Seema Makhija;,citation_author=Pooja Bhagat;,citation_author=Renu Gupta;,citation_author=Utkarsh Sood;,citation_author=Rup Lal;,citation_author=Ravi Toteja;,citation_publication_date=2023-01;,citation_cover_date=2023-01;,citation_year=2023;,citation_issue=1;,citation_doi=10.1007/s11033-022-08015-7;,citation_issn=1573-4978;,citation_pmid=36308581;,citation_volume=50;,citation_language=en-US;,citation_journal_title=Molecular Biology Reports;">
<meta name="citation_reference" content="citation_title=DNA barcoding of southern African mammal species and construction of a reference library for forensic application;,citation_abstract=Combating wildlife crimes in South Africa requires accurate identification of traded species and their products. Diagnostic morphological characteristics needed to identify species are often lost when specimens are processed and customs officials lack the expertise to identify species. As a potential solution, DNA barcoding can be used to identify morphologically indistinguishable specimens in forensic cases. However, barcoding is hindered by the reliance on comprehensive, validated DNA barcode reference databases, which are currently limited. To overcome this limitation, we constructed a barcode library of cytochrome c oxidase subunit 1 and cytochrome b sequences for threatened and protected mammals exploited in southern Africa. Additionally, we included closely related or morphologically similar species and assessed the database’s ability to identify species accurately. Published southern African sequences were incorporated to estimate intraspecific and interspecific variation. Neighbor-joining trees successfully discriminated 94%–95% of the taxa. However, some widespread species exhibited high intraspecific distances ($&amp;amp;amp;gt;$2%), suggesting geographic sub-structuring or cryptic speciation. Lack of reliable published data prevented the unambiguous discrimination of certain species. This study highlights the efficacy of DNA barcoding in species identification, particularly for forensic applications. It also highlights the need for a taxonomic re-evaluation of certain widespread species and challenging genera.;,citation_author=J. R. Baxter;,citation_author=A. Kotze;,citation_author=M. De Bruyn;,citation_author=K. Matlou;,citation_author=K. Labuschagne;,citation_author=M. Mwale;,citation_publication_date=2024-10;,citation_cover_date=2024-10;,citation_year=2024;,citation_issue=10;,citation_doi=10.1139/gen-2023-0050;,citation_issn=0831-2796, 1480-3321;,citation_volume=67;,citation_language=en-US;,citation_journal_title=Genome;">
<meta name="citation_reference" content="citation_title=DNA barcoding of southern African mammal species and construction of a reference library for forensic application;,citation_abstract=Combating wildlife crimes in South Africa requires accurate identification of traded species and their products. Diagnostic morphological characteristics needed to identify species are often lost when specimens are processed and customs officials lack the expertise to identify species. As a potential solution, DNA barcoding can be used to identify morphologically indistinguishable specimens in forensic cases. However, barcoding is hindered by the reliance on comprehensive, validated DNA barcode reference databases, which are currently limited. To overcome this limitation, we constructed a barcode library of cytochrome c oxidase subunit 1 and cytochrome b sequences for threatened and protected mammals exploited in southern Africa. Additionally, we included closely related or morphologically similar species and assessed the database’s ability to identify species accurately. Published southern African sequences were incorporated to estimate intraspecific and interspecific variation. Neighbor-joining trees successfully discriminated 94%–95% of the taxa. However, some widespread species exhibited high intraspecific distances ($&amp;amp;amp;gt;$2%), suggesting geographic sub-structuring or cryptic speciation. Lack of reliable published data prevented the unambiguous discrimination of certain species. This study highlights the efficacy of DNA barcoding in species identification, particularly for forensic applications. It also highlights the need for a taxonomic re-evaluation of certain widespread species and challenging genera.;,citation_author=J. R. Baxter;,citation_author=A. Kotze;,citation_author=M. De Bruyn;,citation_author=K. Matlou;,citation_author=K. Labuschagne;,citation_author=M. Mwale;,citation_publication_date=2024-10;,citation_cover_date=2024-10;,citation_year=2024;,citation_issue=10;,citation_doi=10.1139/gen-2023-0050;,citation_issn=0831-2796, 1480-3321;,citation_volume=67;,citation_language=en-US;,citation_journal_title=Genome;">
<meta name="citation_reference" content="citation_title=Convention on Biological Diversity. Convention on Biological Diversity;,citation_author=undefined CBD;">
<meta name="citation_reference" content="citation_title=Convention on Biological Diversity. Convention on Biological Diversity;,citation_author=CBD;">
<meta name="citation_reference" content="citation_title=A review of over a decade of DNA barcoding in South Africa: A faunal perspective;,citation_abstract=For over a decade, molecular short standardised DNA fragments, termed DNA barcodes, have been developed for species discrimination around the world. As of 2010, the vast majority of barcoding research was biased toward particular taxonomic groups and geographic regions largely because researchers in developed countries were the ones with the resources and capacity to carry out such work. To rectify this, the International Barcode of Life Project was launched with the intent to extend the geographic and taxonomic coverage of the barcode reference library. South Africa committed to this mission in an attempt to catalogue all of its known biodiversity and, possibly, help identify new species. To date, approximately 48 000 South African faunal barcodes are housed in the Barcode of Life Data System (BOLD), which represent only 2.3% of all known South African animal species. Although insects are the best represented in absolute terms, with over 37 000 samples recorded, they are still grossly lacking with just over 1% representation. Much like the global trend, there is a general taxonomic bias, with fish, birds and mammals showing the greatest representation. Moreover, geographic bias is also present, with the Free State province particularly under-represented on BOLD, likely owing to limited human capacity. Although few studies have been published with respect to barcoding, the majority reveal that the cytochrome c oxidase 1 (CO1) gene, used in isolation or in conjunction with other molecular markers, can greatly benefit South African biodiversity research. Several limitations of DNA barcoding are discussed and recommendations specific to South Africa provided.;,citation_author=Jessica M Silva;,citation_author=Sandi Willows-Munro;,citation_publication_date=2016-01;,citation_cover_date=2016-01;,citation_year=2016;,citation_issue=1;,citation_doi=10.1080/15627020.2016.1151377;,citation_issn=1562-7020, 2224-073X;,citation_volume=51;,citation_language=en-US;,citation_journal_title=African Zoology;">
<meta name="citation_reference" content="citation_title=A review of over a decade of DNA barcoding in South Africa: A faunal perspective;,citation_abstract=For over a decade, molecular short standardised DNA fragments, termed DNA barcodes, have been developed for species discrimination around the world. As of 2010, the vast majority of barcoding research was biased toward particular taxonomic groups and geographic regions largely because researchers in developed countries were the ones with the resources and capacity to carry out such work. To rectify this, the International Barcode of Life Project was launched with the intent to extend the geographic and taxonomic coverage of the barcode reference library. South Africa committed to this mission in an attempt to catalogue all of its known biodiversity and, possibly, help identify new species. To date, approximately 48 000 South African faunal barcodes are housed in the Barcode of Life Data System (BOLD), which represent only 2.3% of all known South African animal species. Although insects are the best represented in absolute terms, with over 37 000 samples recorded, they are still grossly lacking with just over 1% representation. Much like the global trend, there is a general taxonomic bias, with fish, birds and mammals showing the greatest representation. Moreover, geographic bias is also present, with the Free State province particularly under-represented on BOLD, likely owing to limited human capacity. Although few studies have been published with respect to barcoding, the majority reveal that the cytochrome c oxidase 1 (CO1) gene, used in isolation or in conjunction with other molecular markers, can greatly benefit South African biodiversity research. Several limitations of DNA barcoding are discussed and recommendations specific to South Africa provided.;,citation_author=Jessica M Da Silva;,citation_author=Sandi Willows-Munro;,citation_publication_date=2016-01;,citation_cover_date=2016-01;,citation_year=2016;,citation_issue=1;,citation_doi=10.1080/15627020.2016.1151377;,citation_issn=1562-7020, 2224-073X;,citation_volume=51;,citation_language=en-US;,citation_journal_title=African Zoology;">
<meta name="citation_reference" content="citation_title=Environmental DNA for tracking waterhole visitation in savanna ecosystems;,citation_author=Maxwell J Farrell;,citation_author=Danny Govender;,citation_author=Mehrdad Hajibabaei;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Environmental DNA for tracking waterhole visitation in savanna ecosystems;,citation_author=Maxwell J Farrell;,citation_author=Danny Govender;,citation_author=Mehrdad Hajibabaei;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Biological identifications through DNA barcodes;,citation_abstract=Although much biological research depends upon species diagnoses, taxonomic expertise is collapsing. We are convinced that the sole prospect for a sustainable identification capability lies in the construction of systems that employ DNA sequences as taxon ’barcodes’. We establish that the mitochondrial gene cytochrome c oxidase I (COI) can serve as the core of a global bioidentification system for animals. First, we demonstrate that COI profiles, derived from the low-density sampling of higher taxonomic categories, ordinarily assign newly analysed taxa to the appropriate phylum or order. Second, we demonstrate that species-level assignments can be obtained by creating comprehensive COI profiles. A model COI profile, based upon the analysis of a single individual from each of 200 closely allied species of lepidopterans, was 100% successful in correctly identifying subsequent specimens. When fully developed, a COI identification system will provide a reliable, cost-effective and accessible solution to the current problem of species identification. Its assembly will also generate important new insights into the diversification of life and the rules of molecular evolution.;,citation_author=Paul D. N. Hebert;,citation_author=Alina Cywinska;,citation_author=Shelley L. Ball;,citation_author=Jeremy R. deWaard;,citation_publication_date=2003-02;,citation_cover_date=2003-02;,citation_year=2003;,citation_issue=1512;,citation_doi=10.1098/rspb.2002.2218;,citation_issn=0962-8452;,citation_pmid=12614582;,citation_volume=270;,citation_language=en-US;,citation_journal_title=Proceedings. Biological Sciences;">
<meta name="citation_reference" content="citation_title=Biological identifications through DNA barcodes;,citation_abstract=Although much biological research depends upon species diagnoses, taxonomic expertise is collapsing. We are convinced that the sole prospect for a sustainable identification capability lies in the construction of systems that employ DNA sequences as taxon ’barcodes’. We establish that the mitochondrial gene cytochrome c oxidase I (COI) can serve as the core of a global bioidentification system for animals. First, we demonstrate that COI profiles, derived from the low-density sampling of higher taxonomic categories, ordinarily assign newly analysed taxa to the appropriate phylum or order. Second, we demonstrate that species-level assignments can be obtained by creating comprehensive COI profiles. A model COI profile, based upon the analysis of a single individual from each of 200 closely allied species of lepidopterans, was 100% successful in correctly identifying subsequent specimens. When fully developed, a COI identification system will provide a reliable, cost-effective and accessible solution to the current problem of species identification. Its assembly will also generate important new insights into the diversification of life and the rules of molecular evolution.;,citation_author=Paul D. N. Hebert;,citation_author=Alina Cywinska;,citation_author=Shelley L. Ball;,citation_author=Jeremy R. deWaard;,citation_publication_date=2003-02;,citation_cover_date=2003-02;,citation_year=2003;,citation_issue=1512;,citation_doi=10.1098/rspb.2002.2218;,citation_issn=0962-8452;,citation_pmid=12614582;,citation_volume=270;,citation_language=en-US;,citation_journal_title=Proceedings. Biological Sciences;">
<meta name="citation_reference" content="citation_title=Advances in environmental DNA monitoring: Standardization, automation, and emerging technologies in aquatic ecosystems;,citation_author=Suxiang Lu;,citation_author=Honghui Zeng;,citation_author=Fan Xiong;,citation_author=Meng Yao;,citation_author=Shunping He;,citation_publication_date=2024-07;,citation_cover_date=2024-07;,citation_year=2024;,citation_issue=7;,citation_doi=10.1007/s11427-023-2493-5;,citation_issn=1674-7305, 1869-1889;,citation_volume=67;,citation_language=en-US;,citation_journal_title=Science China Life Sciences;">
<meta name="citation_reference" content="citation_title=Advances in environmental DNA monitoring: Standardization, automation, and emerging technologies in aquatic ecosystems;,citation_author=Suxiang Lu;,citation_author=Honghui Zeng;,citation_author=Fan Xiong;,citation_author=Meng Yao;,citation_author=Shunping He;,citation_publication_date=2024-07;,citation_cover_date=2024-07;,citation_year=2024;,citation_issue=7;,citation_doi=10.1007/s11427-023-2493-5;,citation_issn=1674-7305, 1869-1889;,citation_volume=67;,citation_language=en-US;,citation_journal_title=Science China Life Sciences;">
<meta name="citation_reference" content="citation_title=DNA Barcoding for Identification and Detection of Species;,citation_abstract=Identification is a very important part of the taxonomy. Since a species represents the basic unit of biological classification, identifying species is important to understand the systematics and the precise phylogenetic position of particular species. In recent years, species identification and delimitation have seen major improvements because of the incorporation of DNA sequence data. This review provides a comprehensive list of commonly employed nuclear and chloroplast regions used for the barcoding of plants.;,citation_author=R Mathur;,citation_author=I Gunwal;,citation_author=N Chauhan;,citation_author=Y Agrawal;,citation_publication_date=2021-09;,citation_cover_date=2021-09;,citation_year=2021;,citation_issue=2;,citation_doi=10.33263/LIANBS112.35423548;,citation_issn=2284-6808;,citation_volume=11;,citation_language=en-US;,citation_journal_title=Letters in Applied NanoBioScience;">
<meta name="citation_reference" content="citation_title=DNA Barcoding for Identification and Detection of Species;,citation_abstract=Identification is a very important part of the taxonomy. Since a species represents the basic unit of biological classification, identifying species is important to understand the systematics and the precise phylogenetic position of particular species. In recent years, species identification and delimitation have seen major improvements because of the incorporation of DNA sequence data. This review provides a comprehensive list of commonly employed nuclear and chloroplast regions used for the barcoding of plants.;,citation_author=R Mathur;,citation_author=I Gunwal;,citation_author=N Chauhan;,citation_author=Y Agrawal;,citation_publication_date=2021-09;,citation_cover_date=2021-09;,citation_year=2021;,citation_issue=2;,citation_doi=10.33263/LIANBS112.35423548;,citation_issn=2284-6808;,citation_volume=11;,citation_language=en-US;,citation_journal_title=Letters in Applied NanoBioScience;">
<meta name="citation_reference" content="citation_title=DNA Barcoding for Identification and Detection of Species;,citation_abstract=Identification is a very important part of the taxonomy. Since a species represents the basic unit of biological classification, identifying species is important to understand the systematics and the precise phylogenetic position of particular species. In recent years, species identification and delimitation have seen major improvements because of the incorporation of DNA sequence data. This review provides a comprehensive list of commonly employed nuclear and chloroplast regions used for the barcoding of plants.;,citation_author=M Mwale;,citation_author=D. L Dalton;,citation_author=R Jansen;,citation_author=M De Bruyn;,citation_author=D Pietersen;,citation_author=P. S Mokgokong;,citation_author=A Kotzé;,citation_publication_date=2021-09;,citation_cover_date=2021-09;,citation_year=2021;,citation_issue=2;,citation_doi=10.33263/LIANBS112.35423548;,citation_issn=2284-6808;,citation_volume=11;,citation_language=en-US;,citation_journal_title=Letters in Applied NanoBioScience;">
<meta name="citation_reference" content="citation_title=DNA Barcoding for Identification and Detection of Species;,citation_abstract=Identification is a very important part of the taxonomy. Since a species represents the basic unit of biological classification, identifying species is important to understand the systematics and the precise phylogenetic position of particular species. In recent years, species identification and delimitation have seen major improvements because of the incorporation of DNA sequence data. This review provides a comprehensive list of commonly employed nuclear and chloroplast regions used for the barcoding of plants.;,citation_author=M Mwale;,citation_author=D. L Dalton;,citation_author=R Jansen;,citation_author=M De Bruyn;,citation_author=D Pietersen;,citation_author=P. S Mokgokong;,citation_author=A Kotzé;,citation_publication_date=2021-09;,citation_cover_date=2021-09;,citation_year=2021;,citation_issue=2;,citation_doi=10.33263/LIANBS112.35423548;,citation_issn=2284-6808;,citation_volume=11;,citation_language=en-US;,citation_journal_title=Letters in Applied NanoBioScience;">
<meta name="citation_reference" content="citation_title=Unlocking the genetic code: Exploring the potential of DNA barcoding for biodiversity assessment;,citation_abstract=DNA barcoding is a crucial method for assessing and monitoring species diversity amidst escalating threats to global biodiversity. I explore DNA barcoding’s potential as a robust and reliable tool for biodiversity assessment. It begins with a comprehensive review of existing literature, delving into the theoretical foundations, methodologies and applications of DNA barcoding. The suitability of various DNA regions, like the COI gene, as universal barcodes are extensively investigated. Additionally, the advantages and limitations of different DNA sequencing technologies and bioinformatics tools are evaluated within the context of DNA barcoding. To evaluate the efficacy of DNA barcoding, diverse ecosystems, including terrestrial, freshwater and marine habitats, are sampled. Extracted DNA from collected specimens undergoes amplification and sequencing of the target barcode region. Comparison of the obtained DNA sequences with reference databases allows for the identification and classification of the sampled organisms. Findings demonstrate that DNA barcoding accurately identifies species, even in cases where morphological identification proves challenging. Moreover, it sheds light on cryptic and endangered species, aiding conservation efforts. I also investigate patterns of genetic diversity and evolutionary relationships among different taxa through the analysis of genetic data. This research contributes to the growing knowledge on DNA barcoding and its applicability for biodiversity assessment. The advantages of this approach, such as speed, accuracy and cost-effectiveness, are highlighted, along with areas for improvement. By unlocking the genetic code, DNA barcoding enhances our understanding of biodiversity, supports conservation initiatives and informs evidence-based decision-making for the sustainable management of ecosystems.;,citation_author=Mohammad Ahmad Ahmad Odah;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=4;,citation_doi=10.3934/molsci.2023016;,citation_issn=2372-0301;,citation_volume=10;,citation_language=en-US;,citation_journal_title=AIMS Molecular Science;">
<meta name="citation_reference" content="citation_title=Unlocking the genetic code: Exploring the potential of DNA barcoding for biodiversity assessment;,citation_abstract=DNA barcoding is a crucial method for assessing and monitoring species diversity amidst escalating threats to global biodiversity. I explore DNA barcoding’s potential as a robust and reliable tool for biodiversity assessment. It begins with a comprehensive review of existing literature, delving into the theoretical foundations, methodologies and applications of DNA barcoding. The suitability of various DNA regions, like the COI gene, as universal barcodes are extensively investigated. Additionally, the advantages and limitations of different DNA sequencing technologies and bioinformatics tools are evaluated within the context of DNA barcoding. To evaluate the efficacy of DNA barcoding, diverse ecosystems, including terrestrial, freshwater and marine habitats, are sampled. Extracted DNA from collected specimens undergoes amplification and sequencing of the target barcode region. Comparison of the obtained DNA sequences with reference databases allows for the identification and classification of the sampled organisms. Findings demonstrate that DNA barcoding accurately identifies species, even in cases where morphological identification proves challenging. Moreover, it sheds light on cryptic and endangered species, aiding conservation efforts. I also investigate patterns of genetic diversity and evolutionary relationships among different taxa through the analysis of genetic data. This research contributes to the growing knowledge on DNA barcoding and its applicability for biodiversity assessment. The advantages of this approach, such as speed, accuracy and cost-effectiveness, are highlighted, along with areas for improvement. By unlocking the genetic code, DNA barcoding enhances our understanding of biodiversity, supports conservation initiatives and informs evidence-based decision-making for the sustainable management of ecosystems.;,citation_author=Mohammad Ahmad Ahmad Odah;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=4;,citation_doi=10.3934/molsci.2023016;,citation_issn=2372-0301;,citation_volume=10;,citation_language=en-US;,citation_journal_title=AIMS Molecular Science;">
<meta name="citation_reference" content="citation_title=BOLD: The Barcode of Life Data System (http://www.barcodinglife.org);,citation_abstract=The Barcode of Life Data System (bold) is an informatics workbench aiding the acquisition, storage, analysis and publication of DNA barcode records. By assembling molecular, morphological and distributional data, it bridges a traditional bioinformatics chasm. bold is freely available to any researcher with interests in DNA barcoding. By providing specialized services, it aids the assembly of records that meet the standards needed to gain BARCODE designation in the global sequence databases. Because of its web-based delivery and flexible data security model, it is also well positioned to support projects that involve broad research alliances. This paper provides a brief introduction to the key elements of bold, discusses their functional capabilities, and concludes by examining computational resources and future prospects.;,citation_author=Sujeevan Ratnasingham;,citation_author=Paul D N Hebert;,citation_publication_date=2007-05;,citation_cover_date=2007-05;,citation_year=2007;,citation_issue=3;,citation_doi=10.1111/j.1471-8286.2007.01678.x;,citation_issn=1471-8278;,citation_pmid=18784790;,citation_volume=7;,citation_journal_title=Molecular Ecology Notes;">
<meta name="citation_reference" content="citation_title=BOLD: The Barcode of Life Data System (http://www.barcodinglife.org);,citation_abstract=The Barcode of Life Data System (bold) is an informatics workbench aiding the acquisition, storage, analysis and publication of DNA barcode records. By assembling molecular, morphological and distributional data, it bridges a traditional bioinformatics chasm. bold is freely available to any researcher with interests in DNA barcoding. By providing specialized services, it aids the assembly of records that meet the standards needed to gain BARCODE designation in the global sequence databases. Because of its web-based delivery and flexible data security model, it is also well positioned to support projects that involve broad research alliances. This paper provides a brief introduction to the key elements of bold, discusses their functional capabilities, and concludes by examining computational resources and future prospects.;,citation_author=Sujeevan Ratnasingham;,citation_author=Paul D N Hebert;,citation_publication_date=2007-05;,citation_cover_date=2007-05;,citation_year=2007;,citation_issue=3;,citation_doi=10.1111/j.1471-8286.2007.01678.x;,citation_issn=1471-8278;,citation_pmid=18784790;,citation_volume=7;,citation_journal_title=Molecular Ecology Notes;">
<meta name="citation_reference" content="citation_title=A DNA-Based Registry for All Animal Species: The Barcode Index Number (BIN) System;,citation_author=Sujeevan Ratnasingham;,citation_author=Paul D. N. Hebert;,citation_editor=Diego Fontaneto;,citation_publication_date=2013-07;,citation_cover_date=2013-07;,citation_year=2013;,citation_issue=7;,citation_doi=10.1371/journal.pone.0066213;,citation_issn=1932-6203;,citation_volume=8;,citation_language=en-US;,citation_journal_title=PLoS ONE;">
<meta name="citation_reference" content="citation_title=A DNA-Based Registry for All Animal Species: The Barcode Index Number (BIN) System;,citation_author=Sujeevan Ratnasingham;,citation_author=Paul D. N. Hebert;,citation_editor=Diego Fontaneto;,citation_publication_date=2013-07;,citation_cover_date=2013-07;,citation_year=2013;,citation_issue=7;,citation_doi=10.1371/journal.pone.0066213;,citation_issn=1932-6203;,citation_volume=8;,citation_language=en-US;,citation_journal_title=PLoS ONE;">
<meta name="citation_reference" content="citation_title=Unveiling vertebrate biodiversity in arid and semi-arid terrestrial ecosystems through eDNA metabarcoding at savanna waterholes;,citation_abstract=Applying environmental DNA (eDNA) metabarcoding to samples from waterholes and their surroundings offers a promising approach for monitoring terrestrial vertebrates in semi-arid and arid ecosystems, such as the southern African savannas. However, minimal guidance exists on key sampling design parameters for terrestrial ecosystems, which can significantly influence species detection. This study investigated the effects of sampled substrate, sampling season, and metabarcoding primer pair on species richness and taxonomic group detection in terrestrial vertebrates, with a focus on mammals, using eDNA samples from waterholes in Botsalano Game Reserve, South Africa. A total of 725 eDNA samples were collected from 94 sampling events across wet and dry seasons, detecting 95 species (45 birds, 42 mammals, 4 amphibians, 3 reptiles, and 1 fish). Sediment samples provided more reliable detection of abundant taxa, whereas water samples had higher detection frequencies of rare taxa. A mixed sampling approach yielded the highest species richness. Sampling during the wet season yielded higher species richness overall, while more mammal species were detected from dry season sampling. Overlap in species detection between the two metabarcoding primers tested was low (47%). We formulate recommendations for future eDNA metabarcoding study designs in similar systems, including remote sampling logistics and discuss potential sources of false positives in eDNA metabarcoding, including (1) secondary eDNA input, (2) incomplete genetic reference databases, and (3) the low genetic resolution of metabarcoding markers.;,citation_author=Tamara Schenekar;,citation_author=Janine Baxter;,citation_author=Irmgard Sedlmayr;,citation_author=Julia Gladitsch;,citation_author=Sibusiso Mahlangu;,citation_author=Monica Mwale;,citation_publication_date=2025-10;,citation_cover_date=2025-10;,citation_year=2025;,citation_doi=10.1101/2025.10.06.680638;,citation_issn=2692-8205;,citation_language=en-US;,citation_publisher=bioRxiv;">
<meta name="citation_reference" content="citation_title=Unveiling vertebrate biodiversity in arid and semi-arid terrestrial ecosystems through eDNA metabarcoding at savanna waterholes;,citation_abstract=Applying environmental DNA (eDNA) metabarcoding to samples from waterholes and their surroundings offers a promising approach for monitoring terrestrial vertebrates in semi-arid and arid ecosystems, such as the southern African savannas. However, minimal guidance exists on key sampling design parameters for terrestrial ecosystems, which can significantly influence species detection. This study investigated the effects of sampled substrate, sampling season, and metabarcoding primer pair on species richness and taxonomic group detection in terrestrial vertebrates, with a focus on mammals, using eDNA samples from waterholes in Botsalano Game Reserve, South Africa. A total of 725 eDNA samples were collected from 94 sampling events across wet and dry seasons, detecting 95 species (45 birds, 42 mammals, 4 amphibians, 3 reptiles, and 1 fish). Sediment samples provided more reliable detection of abundant taxa, whereas water samples had higher detection frequencies of rare taxa. A mixed sampling approach yielded the highest species richness. Sampling during the wet season yielded higher species richness overall, while more mammal species were detected from dry season sampling. Overlap in species detection between the two metabarcoding primers tested was low (47%). We formulate recommendations for future eDNA metabarcoding study designs in similar systems, including remote sampling logistics and discuss potential sources of false positives in eDNA metabarcoding, including (1) secondary eDNA input, (2) incomplete genetic reference databases, and (3) the low genetic resolution of metabarcoding markers.;,citation_author=Tamara Schenekar;,citation_author=Janine Baxter;,citation_author=Irmgard Sedlmayr;,citation_author=Julia Gladitsch;,citation_author=Sibusiso Mahlangu;,citation_author=Monica Mwale;,citation_publication_date=2025-10;,citation_cover_date=2025-10;,citation_year=2025;,citation_doi=10.1101/2025.10.06.680638;,citation_issn=2692-8205;,citation_language=en-US;,citation_publisher=bioRxiv;">
<meta name="citation_reference" content="citation_title=DNA barcoding and traditional taxonomy: An integrated approach for biodiversity conservation;,citation_abstract=Biological diversity is depleting at an alarming rate. Additionally, a vast amount of biodiversity still remains undiscovered. Taxonomy has been serving the purpose of describing, naming, and classifying species for more than 250 years. DNA taxonomy and barcoding have accelerated the rate of this process, thereby providing a tool for conservation practice. DNA barcoding and traditional taxonomy have their own inherent merits and demerits. The synergistic use of both methods, in the form of integrative taxonomy, has the potential to contribute to biodiversity conservation in a pragmatic timeframe and overcome their individual drawbacks. In this review, we discuss the basics of both these methods of biological identification (traditional taxonomy and DNA barcoding), the technical advances in integrative taxonomy, and future trends. We also present a comprehensive compilation of published examples of integrative taxonomy that refer to nine topics within biodiversity conservation. Morphological and molecular species limits were observed to be congruent in $\sim$41% of the 58 source studies. The majority of the studies highlighted the description of cryptic diversity through the use of molecular data, whereas research areas like endemism, biological invasion, and threatened species were less discussed in the literature.;,citation_author=Bhavisha P. Sheth;,citation_author=Vrinda S. Thaker;,citation_publication_date=2017-07;,citation_cover_date=2017-07;,citation_year=2017;,citation_issue=7;,citation_doi=10.1139/gen-2015-0167;,citation_issn=1480-3321;,citation_pmid=28431212;,citation_volume=60;,citation_language=en-US;,citation_journal_title=Genome;">
<meta name="citation_reference" content="citation_title=DNA barcoding and traditional taxonomy: An integrated approach for biodiversity conservation;,citation_abstract=Biological diversity is depleting at an alarming rate. Additionally, a vast amount of biodiversity still remains undiscovered. Taxonomy has been serving the purpose of describing, naming, and classifying species for more than 250 years. DNA taxonomy and barcoding have accelerated the rate of this process, thereby providing a tool for conservation practice. DNA barcoding and traditional taxonomy have their own inherent merits and demerits. The synergistic use of both methods, in the form of integrative taxonomy, has the potential to contribute to biodiversity conservation in a pragmatic timeframe and overcome their individual drawbacks. In this review, we discuss the basics of both these methods of biological identification (traditional taxonomy and DNA barcoding), the technical advances in integrative taxonomy, and future trends. We also present a comprehensive compilation of published examples of integrative taxonomy that refer to nine topics within biodiversity conservation. Morphological and molecular species limits were observed to be congruent in $\sim$41% of the 58 source studies. The majority of the studies highlighted the description of cryptic diversity through the use of molecular data, whereas research areas like endemism, biological invasion, and threatened species were less discussed in the literature.;,citation_author=Bhavisha P. Sheth;,citation_author=Vrinda S. Thaker;,citation_publication_date=2017-07;,citation_cover_date=2017-07;,citation_year=2017;,citation_issue=7;,citation_doi=10.1139/gen-2015-0167;,citation_issn=1480-3321;,citation_pmid=28431212;,citation_volume=60;,citation_language=en-US;,citation_journal_title=Genome;">
<meta name="citation_reference" content="citation_title=National Biodiversity Assessment 2018: The Status of South Africa’s Ecosystems and Biodiversity;,citation_author=A Skowno;,citation_author=C Poole;,citation_author=D Raimondo;,citation_author=K Sink;,citation_author=H Deventer;,citation_author=L Niekerk;,citation_author=L. R Harris;,citation_author=L Smith-Adao;,citation_author=T Zengeya;,citation_author=W Foden;,citation_author=G Midgley;,citation_author=A Driver;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;">
<meta name="citation_reference" content="citation_title=An overview of DNA barcoding of biodiversity in south africa;,citation_author=M. M Kgatla;,citation_author=C Barker;,citation_author=J. R Baxter;,citation_author=A. E Bester-van der Merwe;,citation_author=M Chaisi;,citation_author=A Chakona;,citation_author=M. I Cherry;,citation_author=S. R Daniels;,citation_author=L. H Du Preez;,citation_author=C. R Haddad;,citation_author=P. G Hawkes;,citation_author=C Ho;,citation_author=T. B Hoareau;,citation_author=A Jacobs;,citation_author=K Jacobs;,citation_author=C Janion-Scheepers;,citation_author=B. J Vuuren;,citation_author=R. M Kabongo;,citation_author=T. T Khoza;,citation_author=N. L Khumalo;,citation_author=T Mahlanza;,citation_author=L Makapela;,citation_author=B. G Makhubo;,citation_author=G. W Maneveldt;,citation_author=K Mashego;,citation_author=G Matcher;,citation_author=C. A Matthee;,citation_author=M Mavhunga;,citation_author=J. M Midgley;,citation_author=M Mlambo;,citation_author=D. M Monsanto;,citation_author=R Mthombeni;,citation_author=S. L Murray;,citation_author=S Mynhardt;,citation_author=B-J Nang-Mba;,citation_author=M Ndlovu;,citation_author=S. P Parbhu;,citation_author=undefined Phetla;,citation_author=M Phukuntsi;,citation_author=T. R Pitcher;,citation_author=T Samaai;,citation_author=C. A Simon;,citation_author=K Sink;,citation_author=C. L Sole;,citation_author=G. L Theron;,citation_author=B Asch;,citation_author=M Bank;,citation_author=C. J. M Steenderen;,citation_author=M. H Villet;,citation_author=C. M Visagie;,citation_author=K. A Williams;,citation_author=S Willows-Munro;,citation_author=M. T Sethusa;,citation_author=J. M Silva;,citation_author=M Mwale;,citation_publication_date=2025;,citation_cover_date=2025;,citation_year=2025;">
</head>

<body class="nav-sidebar floating nav-fixed quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top quarto-banner">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="https://www.sanbi.org/" class="navbar-brand navbar-brand-logo">
    <img src="../../imgs/nba-logo-combo.png" alt="" class="navbar-logo light-content">
    <img src="../../imgs/nba-logo-combo.png" alt="" class="navbar-logo dark-content">
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link active" href="../../index.html" aria-current="page"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../contents.html"> 
<span class="menu-text">Contents</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../contents.html">Contents</a></li><li class="breadcrumb-item"><a href="../../content/genetics/genetic-indicators.html">Genetic diversity</a></li><li class="breadcrumb-item"><a href="../../content/genetics/chl_barcoding.html">DNA barcoding in South Africa</a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<header id="title-block-header" class="quarto-title-block default page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../contents.html">Contents</a></li><li class="breadcrumb-item"><a href="../../content/genetics/genetic-indicators.html">Genetic diversity</a></li><li class="breadcrumb-item"><a href="../../content/genetics/chl_barcoding.html">DNA barcoding in South Africa</a></li></ol></nav>
      <h1 class="title">DNA barcoding in South Africa</h1>
            <p class="subtitle lead">Genetic diversity tools</p>
                      </div>
  </div>
    
    <div class="quarto-title-meta-contents">
      <p class="author">
        Mahlatse Kgatla<sup>1</sup> , Janine Baxter<sup>1</sup> <a href="https://orcid.org/0000-0002-6050-1144" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a>, Monica Mwale<sup>1</sup> <a href="https://orcid.org/0000-0003-2180-8917" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a>
      </p>
    </div>
    
   <div class="quarto-title-meta-contents-affiliation">
          <p class="affiliation">
        1. 
        
        South African National Biodiversity Institute
        
      </p>
        </div>

  <div class="quarto-title-meta">

        
      <div>
      <div class="quarto-title-meta-heading">Published</div>
      <div class="quarto-title-meta-contents">
        <p class="date">November 11, 2025</p>
      </div>
    </div>
    
      
    </div>
    
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header sidebar-header-stacked">
      <a href="../../index.html" class="sidebar-logo-link">
      </a>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../contents.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Contents</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/context/about-nba.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">About the NBA</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/context/uses-nba.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Using the NBA</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/context/biodivprof.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">South Africa’s biodiversity profile</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/context/importance.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">The importance of biodiversity</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/key-messages/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Key messages</span></a>
  </div>
</li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="false">
 <span class="menu-text">Integrated findings</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../content/integrated-findings/intf_headline.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Headline indicators</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth3 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/integrated-findings/intf_ecosystems.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem threat status and protection level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/integrated-findings/chl_rle_explained.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">About ecosystem threat status and the RLE</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/integrated-findings/chl_epl_explained.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">About ecosystem protection level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/integrated-findings/intf-species.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Species threat status and protection level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/integrated-findings/intf-species-rli.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Species trends - Red List Index</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/integrated-findings/intf_pressures-condition.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Pressures on biodiversity</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="false">
 <span class="menu-text">Realm findings</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../content/terrestrial/ter_overview.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Terrestrial</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth3 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/terrestrial/ter_ecosystems.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem types</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/terrestrial/ter_pressures-condition.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Pressures on biodiversity</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/terrestrial/ter_ets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem threat status</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/terrestrial/ter_epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem protection level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/terrestrial/ter_ets_x_epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Intersection of ecosystem threat status and protection level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/ter_sp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Terrestrial Species</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../content/freshwater/fw_overview.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Freshwater</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth3 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/freshwater/fw_wet_type.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Wetland Ecosystem Types</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/freshwater/fw_riv_type.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">River Ecosystem Type</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/freshwater/fw_wet_condition.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Wetland Ecological Condition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/freshwater/fw_wet_pressures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Pressures on Wetland Ecosystems</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/freshwater/fw_riv_condition.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">River Ecological Condition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/freshwater/fw_wet_ets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Wetland ecosystem threat status</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/freshwater/fw_riv_ets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">River Ecosystem Threat Status</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/freshwater/fw_wet_epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Wetland ecosystem protection level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/freshwater/fw_riv_epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">River Ecosystem Protection Level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/fw_sp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Freshwater Species</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../content/estuaries/est_overview.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Estuarine</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth3 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/estuaries/est_ecosystem.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem types</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/estuaries/est_habitat.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Add the title of your page here</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/estuaries/est_pressure.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Pressures</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/estuaries/est_condition.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Condition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/estuaries/est_ets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem threat status</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/estuaries/est_epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem protection level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/estuaries/est_ets_epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Intesection of ecosystem threat status and protection level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/est_sp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Estuarine Species</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../content/marine/marine_overview.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Marine</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-8" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-8" class="collapse list-unstyled sidebar-section depth3 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/marine/marine_ecosystems.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem Types</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/marine/marine_pressures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Pressures</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/marine/marine_condition.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecological condition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/marine/marine_ets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem threat status</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/marine/marine_epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem protection level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/marine/marine_ets_epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Intersection of ecosystem threat status and protection levels</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/marine_sp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">The State of South Africa’s Marine Species</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/marine_bird_sp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Marine birds</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/marine/marine_priority_actions.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Knowledge gaps, priority actions and priority areas</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../content/coast/cst_synthesis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Coast</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-9" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-9" class="collapse list-unstyled sidebar-section depth3 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/coast/cst_ecosystems.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem Types</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/coast/cst_pressures-condition.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Pressures and Ecological Condition</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/coast/cst_ets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem Threat Status</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/coast/cst_epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Ecosystem Protection Level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/coast/cst_ets-x-epl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Intersection of Ecosystem Threat Status and Protection Level</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/coast/cst_species.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Species Assessments</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/coast/cst_priorities-actions.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Priority actions and research</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/subantarctic/sub-all_sections.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Subantarctic</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../content/species/sp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Species findings</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-10" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-10" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_plants.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Plants</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_mammals.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Mammals</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_reptiles.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Reptiles</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_birds.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Birds</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_spiders.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Spiders</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_butterflies.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Butterflies</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_amphibians.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Amphibians</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_fwf.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Freshwater Fishes</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_fwfcrabs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Freshwater Crabs</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_anostraca.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Anostraca</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_dragons.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Dragonflies &amp; Damselflies</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_sharks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Sharks (incl. Rays &amp; Chimaeras)</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/species/sp_sparids.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Sparids</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../../content/genetics/genetic-indicators.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Genetic diversity</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-11" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-11" class="collapse list-unstyled sidebar-section depth2 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/genetics/chl-biobank.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Biodiversity Biobanks in South Africa</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/genetics/chl-forensics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Wildlife forensics in South Africa</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/genetics/chl_barcoding.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">DNA barcoding in South Africa</span></a>
  </div>
</li>
      </ul>
  </li>
          <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-12" role="navigation" aria-expanded="false">
 <span class="menu-text">Priority actions, responses and interventions</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-12" role="navigation" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-12" class="collapse list-unstyled sidebar-section depth2 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/integrated-findings/pakg_actions.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Priority actions, responses and interventions</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../content/integrated-findings/pakg_gaps.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Knowledge gaps and research priorities</span></a>
  </div>
</li>
      </ul>
  </li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active" data-toc-expanded="99">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#understanding-south-africas-biodiversity-through-dna-barcoding" id="toc-understanding-south-africas-biodiversity-through-dna-barcoding" class="nav-link active" data-scroll-target="#understanding-south-africas-biodiversity-through-dna-barcoding">Understanding South Africa’s biodiversity through DNA barcoding</a></li>
  <li><a href="#south-african-barcode-of-life-sabol" id="toc-south-african-barcode-of-life-sabol" class="nav-link" data-scroll-target="#south-african-barcode-of-life-sabol">South African Barcode of Life (SABOL)</a></li>
  <li><a href="#south-african-barcoding-efforts-to-date" id="toc-south-african-barcoding-efforts-to-date" class="nav-link" data-scroll-target="#south-african-barcoding-efforts-to-date">South African barcoding efforts to date</a></li>
  <li><a href="#future-directions-and-recommendations" id="toc-future-directions-and-recommendations" class="nav-link" data-scroll-target="#future-directions-and-recommendations">Future directions and recommendations</a></li>
  <li><a href="#recommended-citation" id="toc-recommended-citation" class="nav-link" data-scroll-target="#recommended-citation">Recommended citation</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content quarto-banner-title-block" id="quarto-document-content">





<section id="understanding-south-africas-biodiversity-through-dna-barcoding" class="level2">
<h2 class="anchored" data-anchor-id="understanding-south-africas-biodiversity-through-dna-barcoding">Understanding South Africa’s biodiversity through DNA barcoding</h2>
<p>South Africa’s rich biodiversity spans terrestrial, freshwater, and marine ecosystems, encompassing complex community assemblages of organisms. This diversity is a critical national asset, underpinning strategic objectives across conservation, sustainable resource management, and socio-economic development<span class="citation" data-cites="skownoNationalBiodiversityAssessment2018"><sup><a href="#ref-skownoNationalBiodiversityAssessment2018" role="doc-biblioref">1</a></sup></span>. Understanding and documenting biodiversity is therefore essential for informed decision-making, guiding conservation strategies, sectoral policies, and sustainable use of our natural resources to address challenges such as poverty, inequality and unemployment<span class="citation" data-cites="cbdConventionBiologicalDiversity"><sup><a href="#ref-cbdConventionBiologicalDiversity" role="doc-biblioref">2</a></sup></span>.</p>
<p>Despite this, many South African ecosystems and species remain understudied, even as they face growing threats from habitat loss and fragmentation, overexploitation, climate change, biological invasions, and illegal trade. For numerous species, conservation status remains data deficient or unassessed, complicating efforts to evaluate and mitigate threats (<strong>cite integrated findings on species and ecosystems</strong>). These knowledge gaps are compounded by incomplete taxonomic data, including uncertainties in species classification, identification and distribution across bioregions.</p>
<p>An important tool that addressing these challenges is <strong>DNA barcoding</strong>, which enables rapid and accurate species identification using standardised genetic markers<span class="citation" data-cites="hebertBiologicalIdentificationsDNA2003 odahUnlockingGeneticCode2023"><sup><a href="#ref-hebertBiologicalIdentificationsDNA2003" role="doc-biblioref">3</a>,<a href="#ref-odahUnlockingGeneticCode2023" role="doc-biblioref">4</a></sup></span>. DNA barcoding improves understanding of species diversity across ecosystems and biomes<span class="citation" data-cites="antilDNABarcodingEffective2023 ratnasinghamBOLDBarcodeLife2007"><sup><a href="#ref-antilDNABarcodingEffective2023" role="doc-biblioref">5</a>,<a href="#ref-ratnasinghamBOLDBarcodeLife2007" role="doc-biblioref">6</a></sup></span>, supports the updating of national species checklists, and helps fill taxonomic gaps in priority groups. It complements traditional taxonomy by reliably delimiting species and identofying Operational Taxonomic Units (OTUs), including cryptic, closely related or morphologically similar populations<span class="citation" data-cites="ratnasinghamDNABasedRegistryAll2013 shethDNABarcodingTraditional2017"><sup><a href="#ref-ratnasinghamDNABasedRegistryAll2013" role="doc-biblioref">7</a>,<a href="#ref-shethDNABarcodingTraditional2017" role="doc-biblioref">8</a></sup></span>. DNA barcoding also has practical applications in environmental DNA (eDNA) monitoring of ecosystems, tracking invasive species<span class="citation" data-cites="farrellEnvironmentalDNATracking luAdvancesEnvironmentalDNA2024 schenekarUnveilingVertebrateBiodiversity2025"><sup><a href="#ref-farrellEnvironmentalDNATracking" role="doc-biblioref">9</a>–<a href="#ref-schenekarUnveilingVertebrateBiodiversity2025" role="doc-biblioref">11</a></sup></span> and forensic identification of illegal wildlife trade in South Africa<span class="citation" data-cites="baxterDNABarcodingSouthern2024 mwaleDNABarcodingIdentification2021"><sup><a href="#ref-baxterDNABarcodingSouthern2024" role="doc-biblioref">12</a>,<a href="#ref-mwaleDNABarcodingIdentification2021" role="doc-biblioref">13</a></sup></span>.</p>
<p>DNA barcoding relies on short, standardised DNA regions for identification (e.g., mitochondrial cytochrome oxidase subunit I [<em>COI</em>] for many animals, maturase K [<em>matK</em>] and ribulose-1,5-bisphosphate carboxylase/oxygenase [<em>RbcL</em>] for plants, and Internal Transcribed Spacer [<em>ITS</em>] for fungi<span class="citation" data-cites="mathurDNABarcodingIdentification2021"><sup><a href="#ref-mathurDNABarcodingIdentification2021" role="doc-biblioref">14</a></sup></span>.</p>
</section>
<section id="south-african-barcode-of-life-sabol" class="level2">
<h2 class="anchored" data-anchor-id="south-african-barcode-of-life-sabol">South African Barcode of Life (SABOL)</h2>
<p>The <a href="https://fbip.co.za/about/sanbi-gbif/"><strong>South African Barcode of Life (SABOL)</strong></a> is the national node of the International Barcode of Life Consortium (iBOL). SABOL is funded by the Department of Science, Technology and Innovation (DSI) through the <strong>Foundational Biodiversity Information Programme (FBIP).</strong> SABOL promotes DNA barcoding research, addressing gaps in species reference libraries, and supports the NBA by consolidating scattered biodiversity data into one comprehensive national resource. DNA barcoding strengthens biodiversity assessments, particularly for cryptic and understudied taxa.</p>
<p>Between 2013 and 2022, the FBIP funded 124 DNA barcoding-related projects totaling over R60 million (<a href="#fig-fbip_investment" class="quarto-xref">Figure&nbsp;1</a>). These efforts align with both international and national obligations, including the United Nations Convention on Biological Diversity (CBD), the Intergovernmental Platform on Biodiversity and Ecosystem Services (IPBES), the South Africa’s National Environmental Management: Biodiversity Act (NEM:BA).</p>
<div id="fig-fbip_investment" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-align="center">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-fbip_investment-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="imgs/fbip_investment.png" class="img-fluid quarto-figure quarto-figure-center figure-img" width="600">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-fbip_investment-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1. Funds (in millions of South African Rands) awarded per year to biodiversity research through the Foundational Biodiversity Information Programme (FBIP) between 2013 and 2022. Plot adapted from Mahlatse et al.&nbsp;under review.
</figcaption>
</figure>
</div>
<p>A key component of DNA barcoding is the availability of a comprehensive reference library of expertly identified specimens. The <a href="https://boldsystems.org/"><strong>Barcode of Life Database (BOLD)</strong></a> serves as a trusted repository for these sequences, ensuring that barcodes meet rigorous quality standards and are derived from taxonomically verified specimens. By providing a reliable foundation for species identification, BOLD underpins SABOL and other national barcoding initiatives, enabling accurate biodiversity assessments and monitoring.</p>
</section>
<section id="south-african-barcoding-efforts-to-date" class="level2">
<h2 class="anchored" data-anchor-id="south-african-barcoding-efforts-to-date">South African barcoding efforts to date</h2>
<p>Over the past decade, South Africa has made significant progress in expanding its faunal DNA barcoding database, particularly for insects and other invertebrates. An early review of DNA barcoding reported that South Africa had contributed approximately 48,000 faunal DNA barcodes to BOLD, representing approximately 2.3% of all known species globally, at the time of this review<span class="citation" data-cites="dasilvaReviewDecadeDNA2016"><sup><a href="#ref-dasilvaReviewDecadeDNA2016" role="doc-biblioref">15</a></sup></span>.</p>
<p>Since 2016, contributions to the reference library have grown substantially, with numerous South African researchers and institutions submitting data on indigenous species. As of 28 October 2025, BOLD contains 786,480 sequences (multiple gene sequences for some records), representing <strong>776,512 specimens from South African localities</strong>. Approximately, 93% (725,590) of these records are assigned to 59,606 Barcode Index Numbers (BINs), which represent OTUs. However, only 94,010 specimen records have been identified to species level, representing 16,425 species (<a href="#fig-taxon-bin" class="quarto-xref">Figure&nbsp;2</a>). This highlights ongoing needs for taxonomic capacity and expertise.</p>
<p>This milestone reflects the <strong>collaborative efforts of 284 research institutions</strong>, demonstrating the commitment of South Africa’s biodiversity research community to building a comprehensive DNA barcode species reference library. Despite this progress, challenges remain, including incomplete geographical coverage, taxonomic gaps, limited taxonomic expertise, and the underutilization of barcode data in conservation and management frameworks<span class="citation" data-cites="dasilvaReviewDecadeDNA2016"><sup><a href="#ref-dasilvaReviewDecadeDNA2016" role="doc-biblioref">15</a></sup></span>.</p>
<div id="fig-taxon-bin" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-align="center">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-taxon-bin-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="imgs/taxon-vs-bin.png" class="img-fluid quarto-figure quarto-figure-center figure-img" width="600">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-taxon-bin-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2. BIN (Barcode Index Numbers)to species ratio regression analysis. The relationship between the number of BINs assigned to records and the number of species morphologically identified and assigned to species name, with the orange line indicating the 1:1 ratio (BIN to Species). Plot adapted from Mahlatse et al.&nbsp;under review.
</figcaption>
</figure>
</div>
</section>
<section id="future-directions-and-recommendations" class="level2">
<h2 class="anchored" data-anchor-id="future-directions-and-recommendations">Future directions and recommendations</h2>
<p>A recent gap analysis<span class="citation" data-cites="mahlatse-underreview"><sup><a href="#ref-mahlatse-underreview" role="doc-biblioref">16</a></sup></span> indicates that South Africa’s reference library still has taxonomic and geographic gaps, particularly in priority taxa and under-sampled regions (<a href="#fig-distribution" class="quarto-xref">Figure&nbsp;3</a>). The BIN system relies on multiple sequences per species that capture intraspecific variation across distributions and to improve phylogenetic resolution<span class="citation" data-cites="ratnasinghamDNABasedRegistryAll2013"><sup><a href="#ref-ratnasinghamDNABasedRegistryAll2013" role="doc-biblioref">7</a></sup></span>. To maximize its effectiveness, future sampling should focus on:</p>
<ul>
<li><p><strong>entire species’ ranges</strong>, especially for known species that have not yet been barcode</p></li>
<li><p><strong>sparsely sampled areas</strong>, including biodiversity hotpots and threatened ecosystems</p></li>
<li><p><strong>understudied taxonomic groups</strong>, ensuring priority taxa are adequately represented</p></li>
</ul>
<div id="fig-distribution" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-align="center">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-distribution-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="imgs/distribution.png" class="img-fluid quarto-figure quarto-figure-center figure-img" width="650">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-distribution-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3. Proportional representation of taxon groups with BINs in each South African province. Taxon group ‘Fungi’ is excluded because it had no BIN records on BOLD. Category “Other Animals” are species that do not fit into the outlined taxon groups. Category “Other Organisms” are organisms other than animal, plant or fungi (e.g., Protista). Numbers indicate the number of BINs. Map adapted from Mahlatse et al.&nbsp;under review.
</figcaption>
</figure>
</div>
<p>Currently, a significant number of South African records on BOLD remain private while researchers analyse them (<a href="#fig-public-private" class="quarto-xref">Figure&nbsp;4</a>). While understandable, this can create redundancies and slow the growth of the publicly accessible reference library.</p>
<div id="fig-public-private" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-public-private-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="imgs/private-public.png" class="img-fluid figure-img" width="800">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-public-private-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4. Number of South African records on BOLD broken down by public (published) and private (unpublished) records. Graph adapted from Mahlatse et al.&nbsp;under review.
</figcaption>
</figure>
</div>
<p>To address these challenges, it is recommended that DNA barcoding efforts be <strong>coordinated and collaborative,</strong> ensuring national resources are used efficiently and research gaps are strategically targeted. Establishing a <strong>directory of existing barcoding projects and project leaders</strong>, alongside regular SABOL-led community meetings and conferences, will strengthen communication and collaboration among institutions.</p>
</section>
<section id="recommended-citation" class="level2">
<h2 class="anchored" data-anchor-id="recommended-citation">Recommended citation</h2>
<p>Kgatla, M., Baxter, J., &amp; Mwale, M. 2025. DNA barcoding in South Africa: Genetic diversity tools. National Biodiversity Assessment 2025. South African National Biodiversity Institute. http://nba.sanbi.org.za/.</p>
</section>
<section id="references" class="level2">
<h2 class="anchored" data-anchor-id="references">References</h2>
<div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-skownoNationalBiodiversityAssessment2018" class="csl-entry" role="listitem">
1. Skowno, A. et al. 2018. National <span>Biodiversity Assessment</span> 2018: <span>The Status</span> of <span>South Africa</span>’s <span>Ecosystems</span> and <span>Biodiversity</span>.
</div>
<div id="ref-cbdConventionBiologicalDiversity" class="csl-entry" role="listitem">
2. CBD. Convention on <span>Biological Diversity</span>. <span>Convention</span> on <span>Biological Diversity</span>.
</div>
<div id="ref-hebertBiologicalIdentificationsDNA2003" class="csl-entry" role="listitem">
3. Hebert, P.D.N. et al. 2003. Biological identifications through <span>DNA</span> barcodes. <em>Proceedings. Biological Sciences</em> 270: 313–321. <a href="https://doi.org/10.1098/rspb.2002.2218">https://doi.org/10.1098/rspb.2002.2218</a>
</div>
<div id="ref-odahUnlockingGeneticCode2023" class="csl-entry" role="listitem">
4. Odah, M.A.A. 2023. Unlocking the genetic code: <span>Exploring</span> the potential of <span>DNA</span> barcoding for biodiversity assessment. <em>AIMS Molecular Science</em> 10: 263–294. <a href="https://doi.org/10.3934/molsci.2023016">https://doi.org/10.3934/molsci.2023016</a>
</div>
<div id="ref-antilDNABarcodingEffective2023" class="csl-entry" role="listitem">
5. Antil, S. et al. 2023. <span>DNA</span> barcoding, an effective tool for species identification: A review. <em>Molecular Biology Reports</em> 50: 761–775. <a href="https://doi.org/10.1007/s11033-022-08015-7">https://doi.org/10.1007/s11033-022-08015-7</a>
</div>
<div id="ref-ratnasinghamBOLDBarcodeLife2007" class="csl-entry" role="listitem">
6. Ratnasingham, S. &amp; P.D.N. Hebert. 2007. <span>BOLD</span>: <span>The Barcode</span> of <span>Life Data System</span> (<span class="nocase">http://www.barcodinglife.org)</span>. <em>Molecular Ecology Notes</em> 7: 355–364. <a href="https://doi.org/10.1111/j.1471-8286.2007.01678.x">https://doi.org/10.1111/j.1471-8286.2007.01678.x</a>
</div>
<div id="ref-ratnasinghamDNABasedRegistryAll2013" class="csl-entry" role="listitem">
7. Ratnasingham, S. &amp; P.D.N. Hebert. 2013. A <span>DNA-Based Registry</span> for <span>All Animal Species</span>: <span>The Barcode Index Number</span> (<span>BIN</span>) <span>System</span>. <em>PLoS ONE</em> 8: e66213. <a href="https://doi.org/10.1371/journal.pone.0066213">https://doi.org/10.1371/journal.pone.0066213</a>
</div>
<div id="ref-shethDNABarcodingTraditional2017" class="csl-entry" role="listitem">
8. Sheth, B.P. &amp; V.S. Thaker. 2017. <span>DNA</span> barcoding and traditional taxonomy: <span>An</span> integrated approach for biodiversity conservation. <em>Genome</em> 60: 618–628. <a href="https://doi.org/10.1139/gen-2015-0167">https://doi.org/10.1139/gen-2015-0167</a>
</div>
<div id="ref-farrellEnvironmentalDNATracking" class="csl-entry" role="listitem">
9. Farrell, M.J. et al. Environmental <span>DNA</span> for tracking waterhole visitation in savanna ecosystems.
</div>
<div id="ref-luAdvancesEnvironmentalDNA2024" class="csl-entry" role="listitem">
10. Lu, S. et al. 2024. Advances in environmental <span>DNA</span> monitoring: <span>Standardization</span>, automation, and emerging technologies in aquatic ecosystems. <em>Science China Life Sciences</em> 67: 1368–1384. <a href="https://doi.org/10.1007/s11427-023-2493-5">https://doi.org/10.1007/s11427-023-2493-5</a>
</div>
<div id="ref-schenekarUnveilingVertebrateBiodiversity2025" class="csl-entry" role="listitem">
11. Schenekar, T. et al. 2025. Unveiling vertebrate biodiversity in arid and semi-arid terrestrial ecosystems through <span class="nocase">eDNA</span> metabarcoding at savanna waterholes. 2025.10.06.680638. <a href="https://doi.org/10.1101/2025.10.06.680638">https://doi.org/10.1101/2025.10.06.680638</a>
</div>
<div id="ref-baxterDNABarcodingSouthern2024" class="csl-entry" role="listitem">
12. Baxter, J.R. et al. 2024. <span>DNA</span> barcoding of southern <span>African</span> mammal species and construction of a reference library for forensic application. <em>Genome</em> 67: 378–391. <a href="https://doi.org/10.1139/gen-2023-0050">https://doi.org/10.1139/gen-2023-0050</a>
</div>
<div id="ref-mwaleDNABarcodingIdentification2021" class="csl-entry" role="listitem">
13. Mwale, M. et al. 2021. <span>DNA Barcoding</span> for <span>Identification</span> and <span>Detection</span> of <span>Species</span>. <em>Letters in Applied NanoBioScience</em> 11: 3542–3548. <a href="https://doi.org/10.33263/LIANBS112.35423548">https://doi.org/10.33263/LIANBS112.35423548</a>
</div>
<div id="ref-mathurDNABarcodingIdentification2021" class="csl-entry" role="listitem">
14. Mathur, R. et al. 2021. <span>DNA Barcoding</span> for <span>Identification</span> and <span>Detection</span> of <span>Species</span>. <em>Letters in Applied NanoBioScience</em> 11: 3542–3548. <a href="https://doi.org/10.33263/LIANBS112.35423548">https://doi.org/10.33263/LIANBS112.35423548</a>
</div>
<div id="ref-dasilvaReviewDecadeDNA2016" class="csl-entry" role="listitem">
15. Da Silva, J.M. &amp; S. Willows-Munro. 2016. A review of over a decade of <span>DNA</span> barcoding in <span>South Africa</span>: A faunal perspective. <em>African Zoology</em> 51: 1–12. <a href="https://doi.org/10.1080/15627020.2016.1151377">https://doi.org/10.1080/15627020.2016.1151377</a>
</div>
<div id="ref-mahlatse-underreview" class="csl-entry" role="listitem">
16. Kgatla, M.M. et al. 2025. An overview of DNA barcoding of biodiversity in south africa.
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp("https:\/\/nba\.sanbi\.org\.za\/");
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
            // target, if specified
            link.setAttribute("target", "_blank");
            if (link.getAttribute("rel") === null) {
              link.setAttribute("rel", "noopener");
            }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<div style=" text-align: center; max-width: 260px; line-height: 1.2; font-size: 0.8rem; margin-left: 6em">
<h4 style="margin-bottom: 0.3em; font-size: 1rem; font-weight: 900;">
<a href="../../content/context/about-nba.html" style="color: inherit; text-decoration: none;">About the NBA</a>
</h4>
<p style="margin-top: 0; font-size = 1">
The National Biodiversity Assessment (NBA) is a collaborative effort to synthesise the best available science on the status of South Africa’s biodiversity.
</p>
</div>
</div>   
    <div class="nav-footer-center">
<div style="text-align: center; line-height: 1.6;">
<h4 style="margin-bottom: 0.3em; font-size: 1rem; font-weight: 400;">
Important links
</h4>
<p><a href="../../content/context/acknowledgementsnba2025.html">Acknowledgements</a><br> <a href="../../content/context/cite-nba.html">How to cite</a><br> <a href="../../content/context/whatsnew-nba.html">What’s new</a><br> <a href="../../content/context/glossary.html">Terminology</a><br> <a href="../../content/context/disclaimerterms.html">Disclaimer</a></p>
</div>
</div>
    <div class="nav-footer-right">
<div style="text-align: center; line-height: 1.4;">
  <h4 style="margin-bottom: 0.3em; font-size: 1rem; font-weight: 400;">Contact</h4>
  <p style="margin: 0.2em 0;">
    Email the National Biodiversity Assessment Unit:<br>
    <a href="mailto:nba@sanbi.org.za" style="color: inherit; text-decoration: none;">nba@sanbi.org.za</a>
  </p>
  <div style="margin-top: 0.5em; font-size: 0.85rem;">
    <a href="https://creativecommons.org/licenses/by-nc/4.0/" target="_blank" rel="noopener noreferrer">
      <img src="../../imgs/by-nc.svg" alt="Creative Commons BY-NC 4.0 license" style="height: 31px; border: none;">
    </a>
    <a href="https://github.com/SANBI-NBA" target="_blank" rel="noopener noreferrer">
      <img src="../../imgs/github-mark.svg" alt="NBA on GitHub" style="height: 31px; border: none;">
    </a>        
  </div>
  <p></p>
</div>
</div>
  </div>
</footer>
<script>
  document.addEventListener("DOMContentLoaded", () => {
      document.querySelectorAll("figcaption").forEach(e => {
          const text = e.innerHTML.trim();
          // Only proceed if it starts with "Figure" or "Table"
          if (/^(Figure|Table)\b/.test(text)) {
            const match = text.match(/^([^<]*?\.)\s*(.*)$/s); // Match up to first period before any tag
          if (match) {
            const strongPart = `<strong>${match[1]}</strong>`;
            const rest = match[2];
            e.innerHTML = `${strongPart} ${rest}`;
          }
        }
      });
  });
</script>   




</body></html>